logitrail-woocommerce.php

```diff
@@ -350,69 +350,69 @@ public static function handle_product_import() {
 	}
 
 	function logitrail_create_product($post_id) {
-		global $woocommerce;
-
-		$settings = get_option('woocommerce_logitrail_shipping_settings');
+        if ( get_post_status ( $post_id ) == 'publish' ) {
```

Comment:
You could've just run return if the status is not published. Would've saved from pushing all code inside the if.